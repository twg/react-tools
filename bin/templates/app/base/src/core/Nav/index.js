import React from 'react'
import { Link } from 'react-router-dom'

const Nav = () => (
  <ul>
    <li>
      <Link to="/about">About</Link>
    </li>
    <li>
      <Link to="/users">Fetching data</Link>
    </li>
  </ul>
)

export default Nav
