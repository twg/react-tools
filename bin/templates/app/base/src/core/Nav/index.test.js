/* eslint-env jest */

import React from 'react'
import { MemoryRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'

import Nav from './'

describe('Nav', () => {
  it('matches snapshot', () => {
    const nav = (
      <MemoryRouter>
        <Nav />
      </MemoryRouter>
    )
    let wrapper = renderer.create(nav).toJSON()
    expect(wrapper).toMatchSnapshot()
  })
})
