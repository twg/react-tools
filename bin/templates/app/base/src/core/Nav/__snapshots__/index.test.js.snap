// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav matches snapshot 1`] = `
<ul>
  <li>
    <a
      href="/about"
      onClick={[Function]}
    >
      About
    </a>
  </li>
  <li>
    <a
      href="/users"
      onClick={[Function]}
    >
      Fetching data
    </a>
  </li>
</ul>
`;
